io/FV3GFS_io.F90

```diff
@@ -1501,7 +1501,9 @@ subroutine sfc_prop_restart_read (Sfcprop, Atm_block, Model, fv_domain, warm_sta
 !$omp parallel do default(shared) private(nb, ix)
       do nb = 1, Atm_block%nblks
         do ix = 1, Atm_block%blksz(nb)
-          Sfcprop(nb)%zorlw(ix) = Sfcprop(nb)%zorl(ix) !--- compute zorlw from existing variables
+          if (Sfcprop(nb)%landfrac(ix) < one .and. Sfcprop(nb)%fice(ix) < one) then
+          Sfcprop(nb)%zorlw(ix) = min(Sfcprop(nb)%zorl(ix), 0.317e-2)
```

Comment:
@SMoorthi-emc  Moorthi, how is this 0.317e-2 number decided ?

Reply:
Fanglin,
   I used "z0s_max=.317e-2_kp" from sfc_diff.f.
Nevertheless, this loop is used only when the initial condition does not have zorlw and it is recomputed in sfc_diff.f when not coupled with waves and replaced by wave produced z0 when coupled (except over lakes).

Reply:
Well, I may have messed up the units.  It may have to be multiplied by 100.

Reply:
Jun, is it ok to make this unit correction now or is it too late?
Thanks
Moorthi